app/actions/AccountMixerActions.js

```diff
@@ -102,8 +102,7 @@ export const runAccountMixer = ({
       const mixerStreamer = await dispatch(
         unlockAcctAndExecFn(
           passphrase,
-          changeAccount,
-          null,
+          new Array(changeAccount),
```

Comment:
This is creating an array with number of `changeAccount` positions.

Ex: if change account equals 5, it will create an array with 5 positions and arr[0] will be undefined.

Use brakets `[ ]` instead of `new Array`